regress/pesterTests/SSHDConfig.tests.ps1

```diff
@@ -211,6 +212,27 @@ Match User matchuser
             $tC++
         }
 
+        It "$tC.$tI-Test UnusedConnectionTimeout"  -skip:$skip {
+            #Run
+            Start-SSHDTestDaemon -WorkDir $opensshbinpath -Arguments "-d -f $sshdConfigPath -E $sshdlog" -Port $port
+            Add-UserToLocalGroup -UserName $localuser1 -Password $password -GroupName $allowGroup1
+
+            # Start SSH process with Remote Forwarding Option to create a connection that doesn't prevent "Unused Connection Timeout"
+            $p = Start-Process -FilePath ssh -ArgumentList "-p $port -N -T -R 35000 $localuser1@$server" -PassThru
+            Wait-Process $p.Id -Timeout 10 -ErrorAction SilentlyContinue -ErrorVariable timeouted
```

Comment:
We usually put a wait action in a retry loop, so that the test will finish quicker if the timeout is around the expected 2 minutes (seconds?).  The retry loop will retry to up to a maximum time to wait.  Also we should try to let the test finish as soon as possible, is the `2` 2 minutes or seconds?

Reply:
It's 2 seconds